packages/page-migration/src/Migration/index.tsx

```diff
@@ -131,13 +105,13 @@ function MigrationApp ({ className }): React.ReactElement {
           Select your Dock account. If it isn't there follow <a href="https://docs.dock.io/dock-token/dock-token-migration/adding-account-to-the-dock-browser-wallet" target="_blank">these instructions</a>.
         </li>
         <li>
-        Enter your cheqd account manually or connect a web browser extension wallet. Connecting an extension will allow us to confirm that the tokens are going to the cheqd account that you control.
+          Connect your cheqd wallet or enter your cheqd account manually. Connecting a Leap wallet will allow us to confirm that the tokens are going to the cheqd account that you control. 
```

Comment:
I noticed on line 183 you changed from `kelp` to `wallet extension`. I know we decided we want to push people to use Leap but I just want to make sure this is the ok text

Reply:
yeah the push for leap is more towards the other migration text not on this page, this page should support any browser wallet that has cheqd